scripts/chain-prep.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#set -eux
+killall node
+ganache-cli -u 0 -u 1 -u 2 -m 'cook mango twist then skin sort option civil have still rather guilt' &
+git clone https://github.com/althea-mesh/simple-bidirectional-erc20-channel contract
```

Comment:
Just a thought - Wouldn't we want to maintain a submodule with the contract, or we want to always clone a current master?

Reply:
I don't really care for submodules, but we could do that. it's not great when another repo can break a different repo but it's the best way to make sure things don't get decoupled, with a pin updating the pin can become a technical debt filled pain versus something you have to face with every push.

Reply:
@jkilpatr I see your point. A good strategy for this might be a pin + a additional job on travis that would checkout master branch of a contract and run the tests again. So this way we could know if the 'current release' (a pin) would pass, and the 'next release' (master branch) passes too (or fails). Similar to what we do with rust stable/beta/nightly. This way we could even trigger that separate job from the contract repo on every commit to master on the contract repo.